### PR TITLE
fix: #2622 drawer-end makes horizontal scrollbar permanently visible

### PR DESCRIPTION
**Pull request by binjuhor**
+ Updated src/components/styled/drawer.css to add overflow-hidden

### CODE REVIEW

File: src/components/styled/drawer.css [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .drawer {
   width: 100%;
   &-side {
+    @apply overflow-hidden;
```
Comment (saadeghi) [inline src/components/styled/drawer.css:4]: overflow hidden is not the solution. if sidebar content is long, it should overflow.

**Review by saadeghi — CHANGES_REQUESTED** [review]
What is the problem? can you give a screenshot?

File: src/components/styled/drawer.css (hunk 1 above)
Comment (binjuhor) [reply src/components/styled/drawer.css:4]: This is the problem .
image

>overflow hidden is not the solution. if sidebar content is long, it should overflow. 

yeah I see, so my solution is `@apply overflow-x-hidden;`  @saadeghi

**Comment by lucasrivoiro** [comment]
I have same issue here, but applying overflow-x-hidden to drawer-side create another issue for me. Applying overflow-x-hidden turns the entire window vertically scrollable, but horizontally fix the issue.

**Comment by binjuhor** [comment]
@lucasrivoiro Can you share me the screenshot? I think the window vertical should scrollable because the content can be so long like this

https://github.com/saadeghi/daisyui/assets/4584599/15982d8e-3d5a-44cb-ba45-f069f040ebc7

**Comment by LanceMoe** [comment]
At Windows, Or set the scroll bar to always show in macOS, the Drawer will show a scroll bar.
Before version 4.4.14, it worked well.
image

**Comment by brendoarrudaa** [comment]
Any chance of error being adjusted? I'm in a project that after upgrading from version 3 to 4 broke the entire drawer end of the page

**Comment by cloudhoy** [comment]
@brendoarrudaa for now, you can fix this by adding `overflow-x-hidden`  to element with the 'drawer-side' class:
```
<div class="drawer-side overflow-x-hidden">
  {/* Sidebar content here */}
</div>
```

**Comment by binjuhor** [comment]
@cloudhoy Yeah that also my solution, but I don't know why this PR has not merged yet.

**Comment by brendoarrudaa** [comment]
@cloudhoy  I share the same feeling, it just needs to be approved to resolve the bugs,
@saadeghi Any chance of being approved?

**Pull request merged**